app/data_providers/slate_providers/recommended_reads_slate_provider.py

```diff
@@ -14,3 +31,26 @@ def headline(self) -> str:
     @property
     def subheadline(self) -> str:
         return 'Curated by Pocket'
+
+    async def rank_corpus_items(
+            self,
+            items: List[CorpusItemModel],
+            *args,
+            **kwargs,
+    ) -> List[CorpusItemModel]:
+        """
+        :param items: Candidate corpus items
+        :return: Ranks items based on Thompson sampling if enable_thompson_sampling is True.
+        """
+        if kwargs.get('enable_thompson_sampling'):
+            metrics = await self.corpus_engagement_provider.get(
+                self.recommendation_surface_id, self.configuration_id, items)
+
+            items = thompson_sampling(
+                recs=items,
+                metrics=metrics,
+                trailing_period=7,  # With few new items/day and relatively many impressions, a low period is sufficient
+                default_alpha_prior=7,   # beta * P95 item CTR for this slate (0.7%)
+                default_beta_prior=850)  # 5% of average daily item impressions for this slate
```

Comment:
i would not mind impression capping here.  if slate turns over once a day, I think there is some risk of items not circulating

Reply:
Created [TDP-228](https://getpocket.atlassian.net/browse/TDP-228) to track.